dev-portal/package.json

```diff
@@ -1,25 +1,33 @@
 {
```

Comment:
How many deprecation / npm security warns / etc. are left on `npm install`?

Reply:
Zero. We've got one peer dependency warning, but it doesn't seem to be affecting anything.

Reply:
Sweet!